build.gradle

```diff
@@ -48,20 +48,14 @@ repositories {
 
 dependencies {
     compile gradleApi()
-    compile 'com.android.tools.build:gradle:1.5.0'
-    compile 'com.google.code.gson:gson:2.5'
+    compile 'com.android.tools.build:gradle:2.1.0'
```

Comment:
I do not there are any breaking changes. All my github projects use the latest and I have had no issues upgrading android libraries and applications to use the latest.